ExportAs/ExportAs.py

```diff
@@ -38,32 +38,60 @@ def __init__(self, scriptedPlugin):
     AbstractScriptedSubjectHierarchyPlugin.__init__(self, scriptedPlugin)
 
     self.exportAsAction = qt.QAction(f"Export as...", scriptedPlugin)
+    self.exportTransformedAsAction = qt.QAction(f"Export transformed as...", scriptedPlugin)
     self.menu = qt.QMenu("Plugin Menu")
+    self.transformedMenu = qt.QMenu("Transformed Plugin Menu")
     self.exportAsAction.setMenu(self.menu)
+    self.exportTransformedAsAction.setMenu(self.transformedMenu)
 
   #
   # item context menus are what happens when you right click on a selected line
   #
   def itemContextMenuActions(self):
     """the actions that could be shown for any of this plugins items"""
-    return [self.exportAsAction]
+    return [self.exportAsAction, self.exportTransformedAsAction]
 
   def showContextMenuActionsForItem(self, itemID):
     """Set actions visible that are valid for this itemID"""
     pluginHandlerSingleton = slicer.qSlicerSubjectHierarchyPluginHandler.instance()
     subjectHierarchyNode = pluginHandlerSingleton.subjectHierarchyNode()
     self.exportAsAction.visible = True
     self.exportAsAction.enabled = False
+    self.exportTransformedAsAction.visible = False
+    self.exportTransformedAsAction.enabled = False
     self.menu.clear()
+    self.transformedMenu.clear()
     associatedNode = subjectHierarchyNode.GetItemDataNode(itemID)
     if associatedNode is not None and associatedNode.IsA("vtkMRMLStorableNode"):
         self.exportAsAction.enabled = True
         writerExtensions = slicer.app.coreIOManager().fileWriterExtensions(associatedNode)
         for writerFormat in writerExtensions:
           a = self.menu.addAction(writerFormat)
-          a.connect("triggered()", lambda writerFormat=writerFormat : self.export(associatedNode, writerFormat))
+          a.connect("triggered()", lambda writerFormat=writerFormat : self.export(associatedNode, writerFormat, False))
+    
+    if associatedNode is not None and associatedNode.IsA("vtkMRMLStorableNode") and associatedNode.GetTransformNodeID():
```

Comment:
Bit of a nitpick here, but it should check if the node is a `vtkMRMLTransformableNode` here, since that's the level of inheritance where the `GetTransformNodeID` method is added.  In Slicer core all storable nodes are transformable, but an extension could for some reason inherit from `vtkMRMLStorableNode`.

See:
http://apidocs.slicer.org/master/classvtkMRMLTransformableNode.html

Reply:
Do you mean to also check if associatedNode is a vtkMRMLTransformableNode?

Reply:
Yes, just change the text in the `IsA` call.

Reply:
Is this ready to be commited and merged?

Reply:
No, it still needs to be updated.  Do you just want me to do it?

Reply:
Sure.

Reply:
@muratmaga it looks like there are some other unresolved issues.  @arpinerap still working on it?

Reply:
Sorry people, COVID has take me out of order for a couple of weeks! I´ll review changes requested and try to fix it to close issue.

Reply:
Ok, @pieper , I´ve seen your commit, thanks so much...

Reply:
Yikes, COVID?!?!  Welcome back!

Reply:
Thanks for everything!!!